app/models/Screening.scala

```diff
@@ -0,0 +1,33 @@
+package models
+
+import java.time.LocalDateTime
+
+case class Screening(movie: Movie, screeningRoom: ScreeningRoom, startTime: LocalDateTime) {
+
+  def screeningRoomPlaces: Array[Array[String]] = realScreeningRoomPlaces
+  
+  private val realScreeningRoomPlaces: Array[Array[String]] = Array.ofDim[String](11, 10)
+
+  def makeReservation(tickets: Seq[Ticket]): Unit =
+    for (ticket <- tickets) {
+      realScreeningRoomPlaces(ticket.rowNumber).update(ticket.seatNumberInRow, "BUSY")
+      println("one place reserved")
+    }
+
+  def initializeScreeningRoomPlaces(): Unit = {
```

Comment:
do you think that mixing code with test data with production code is ok?

Reply:
No, it is not. I forgot to remove few println() that I used for testing purposes.